org.eclipse.jdt.ls.tests/projects/eclipse/hello/src/java/Highlight.java

```diff
@@ -0,0 +1,12 @@
+package java;
```

Comment:
Do you mind moving it to `org.sample`? Turns out choosing the `java` package was a terrible idea in the 1st place, as it's a prohibited package name (Try running your class from the `java` package)